packages/block-editor/src/components/global-styles/dimensions-panel.js

```diff
@@ -707,6 +726,23 @@ export default function DimensionsPanel( {
 					/>
 				</ToolsPanelItem>
 			) }
+			{ showHeightControl && (
+				<ToolsPanelItem
+					hasValue={ hasHeightValue }
+					label={ __( 'Height' ) }
+					onDeselect={ resetHeightValue }
+					isShownByDefault={
+						defaultControls.height ?? DEFAULT_CONTROLS.height
+					}
+					panelId={ panelId }
+				>
+					<HeightControl
+						label={ __( 'Height' ) }
+						value={ heightValue }
+						onChange={ setHeightValue }
```

Comment:
Similar to question above. `setMinHeightValue` removes any applied aspect ratio. Is that required here too?

And if so, does it need handling in `gutenberg_render_dimensions_support`?